### PR TITLE
content fixes

### PR DESCRIPTION
**Pull request by oliwiapolec**
## 🚀 Links

- [Feature branch](https://developers.labs.livechat.com/docs/feature/contentfixes)

## 📓 Description

Small content fixes:
- https://developers.livechat.com/docs/management/configuration-api/#update-auto-access changed groups to "groups" in sample response
- https://developers.livechat.com/docs/management/configuration-api/#list-webhooks corrected request sample
- updated link to the Console in Monetization Overview
- added "name" field in the response for List Agents method in 3.3 and 3.4 configuration API

### Content
- Proofreading/content fixes

### CODE REVIEW

File: content/monetization/index.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ LiveChat Platform provides you with a variety of monetization tools. Depending o
 
 ## Paid Apps
 
-You can create a paid app right within the [Developers Console](/console/). It's the simplest way to generate revenue from apps installed by LiveChat customers.
+You can create a paid app right within the [Developers Console](https://developers.livechat.com/console/). It's the simplest way to generate revenue from apps installed by LiveChat customers.
```
Comment (ale-kacperczyk) [inline content/monetization/index.mdx:15]: Please use {{DEVELOPER_CONSOLE_URL}}

Comment (oliwiapolec) [inline content/monetization/index.mdx:15]:
```suggestion
You can create a paid app right within the {{DEVELOPER_CONSOLE_URL}}. It's the simplest way to generate revenue from apps installed by LiveChat customers.
```

Comment (oliwiapolec) [reply content/monetization/index.mdx]: done :D

**Review by ale-kacperczyk — APPROVED** [review]

File: payloads/management/v3.3/configuration-api/responses/agents/listAgents.json [hunk 2]
```diff
@@ -1,16 +1,24 @@
...
       "daily_summary",
       "weekly_summary"
     ]
   },
   {
+    "avatar_path": "cdn.livechatinc.com/avatars/2.png",
     "id": "adam@example.com",
     "job_title": "Support Hero (Newbie)",
+    "login_status": "accepting chats",
+    "name": "Adam",
```
Comment (gluszczyk) [inline payloads/management/v3.3/configuration-api/responses/agents/listAgents.json:20]: Jones or Brown pls, not some Adam 😉

File: payloads/management/v3.4/configuration-api/responses/agents/listAgents.json [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
       "daily_summary",
       "weekly_summary"
     ]
   },
   {
+    "avatar_path": "cdn.livechatinc.com/avatars/2.png",
     "id": "adam@example.com",
     "job_title": "Support Hero (Newbie)",
+    "login_status": "accepting chats",
+    "name": "Adam",
```
Comment (gluszczyk) [inline payloads/management/v3.4/configuration-api/responses/agents/listAgents.json:20]: as above

File: payloads/management/v3.4/configuration-api/responses/agents/listAgents.json [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
+    "role": "viceowner",
     "max_chats_count": 2,
     "summaries": [
       "weekly_summary"
     ]
   }
-]
+]
```
Comment (gluszczyk) [inline payloads/management/v3.4/configuration-api/responses/agents/listAgents.json:27]: 😨

File: payloads/management/v3.3/configuration-api/responses/agents/listAgents.json (hunk 2 above)
Comment (oliwiapolec) [reply payloads/management/v3.3/configuration-api/responses/agents/listAgents.json:20]: sure, I named them such as they had names like that in the ids 😅

File: payloads/management/v3.3/configuration-api/responses/agents/listAgents.json [hunk 5]
```diff
@@ -1,16 +1,24 @@
...
+    "role": "owner",
     "max_chats_count": 5,
     "summaries": [
       "daily_summary",
       "weekly_summary"
     ]
   },
   {
+    "avatar_path": "cdn.livechatinc.com/avatars/2.png",
     "id": "adam@example.com",
```
Comment (oliwiapolec) [inline payloads/management/v3.3/configuration-api/responses/agents/listAgents.json:17]:
```suggestion
    "id": "jones@example.com",
```

File: payloads/management/v3.3/configuration-api/responses/agents/listAgents.json (hunk 2 above)
Comment (oliwiapolec) [inline payloads/management/v3.3/configuration-api/responses/agents/listAgents.json:20]:
```suggestion
    "name": "Jones",
```

File: payloads/management/v3.4/configuration-api/responses/agents/listAgents.json [hunk 6]
```diff
@@ -1,19 +1,27 @@
...
+    "role": "owner",
     "max_chats_count": 5,
     "summaries": [
       "daily_summary",
       "weekly_summary"
     ]
   },
   {
+    "avatar_path": "cdn.livechatinc.com/avatars/2.png",
     "id": "adam@example.com",
```
Comment (oliwiapolec) [inline payloads/management/v3.4/configuration-api/responses/agents/listAgents.json:17]:
```suggestion
    "id": "jones@example.com",
```

File: payloads/management/v3.4/configuration-api/responses/agents/listAgents.json (hunk 3 above)
Comment (oliwiapolec) [inline payloads/management/v3.4/configuration-api/responses/agents/listAgents.json:20]:
```suggestion
    "name": "Jones",
```

**Review by gluszczyk — APPROVED** [review]

**Pull request merged**